app/settings.py

```diff
@@ -49,6 +49,9 @@ def get_env_or_fail(key):
 EQ_SECRETS_FILE = os.getenv("EQ_SECRETS_FILE", "secrets.yml")
 EQ_KEYS_FILE = os.getenv("EQ_KEYS_FILE", "keys.yml")
 
+EQ_PUBLISHER_BACKEND = os.getenv("EQ_PUBLISHER_BACKEND")
+EQ_PUB_SUB_TOPIC_ID = os.getenv("EQ_PUB_SUB_TOPIC_ID")
```

Comment:
The app could publish to more than one topic. This variable name should be specific to its use. The Terraform variable is `eq-fulfilment-topic`, so this could be `EQ_FULFILMENT_TOPIC`.

In the Terraform we refer to this as "name" rather than "id". We should be consistent.

Reply:
Have updated the variable name. However, as for ID/Name, the correct term here would be "id" as topic name is referred to as the full topic path in GCP. But happy to use name if we want to be consistent with existing Terraform implementation but also happy to updated terraform implementation to id. Let me know you preference.